Autopilot/Inc/SensorFusion.hpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * Sensor Fusion - Converts raw sensor data into human readable formatted structs
+ * Author: Lucy Gong, Dhruv Rawat, Anthony Bertnyk
+ */
+
+#include "AttitudeDatatypes.hpp"
+#include "fetchSensorMeasurementsMode.hpp"
+#include "arm_math.h"
+
+#ifndef SENSORFUSION_HPP
+#define SENSORFUSION_HPP
+typedef struct {
```

Comment:
I don't think `typedef` is needed here. With C++ structs there really is no need